### ISSUE TITLE
Train_val_test spilt

### ISSUE THREAD

**Issue by gunjannaik** [naomifridman/Unet_Brain_tumor_segmentation#1]
Hi,

I was going through your code and I saw that you have split the training data into train and test. Whereas, you have use the validation data as it is. So, when we will test, the results will be very good on test, as you have taken test cases from train set. 

I think, it will be better to take some part of validation as test set. Please provide your thoughts on this.
